### PR TITLE
feat: show stats

### PR DESCRIPTION
**Pull request by mdjastrzebski**
### Summary

Display usage stats: response time and used tokens.

Control:
* `--stats` CLI option
* `/stats [on|off]` command

### Test plan

User testing.

### CODE REVIEW

File: src/output.ts [hunk 1]
```diff
@@ -61,3 +73,12 @@ function extractErrorMessage(error: unknown) {
 
   return 'Unknown error';
 }
+
+function formatStats(stats: ResponseStats) {
+  const parts = [
+    `time: ${(stats.responseTime / 1000).toFixed(1)} s`,
+    `tokens: ${stats.prompt_tokens} in + ${stats.completion_tokens} out`,
```
Comment (Q1w1N) [inline src/output.ts:88]: maybe we could also add a sum of that? QoL improvement to avoid checking stats with calc opened :P

**Comment by Q1w1N** [comment]
I tested that, and it would be really awesome if stats could be also displayed retroactively. So if all of previous messages could get stats next to them as well (not sure if easily achievable, but would be amazing)

File: src/output.ts (hunk 1 above)
Comment (mdjastrzebski) [reply src/output.ts:88]: Why would you like to have a sum? input and output tokes have differnt price, etc

Comment (mdjastrzebski) [reply src/output.ts:88]: In future we could think about adding "cost" state, but we would have to encode the pricing of each model

**Comment by mdjastrzebski** [comment]
Redisplaying all previous messages, would involve modifying the previous terminal lines (potentially also above the scroll). Not sure how difficult is that. If you want, we could do it later as a nice-to-have feature

**Review by Q1w1N — APPROVED** [review]

**Pull request merged**